pandas/core/base.py

```diff
@@ -148,10 +150,20 @@ class SpecificationError(Exception):
 
 class SelectionMixin:
     """
-    mixin implementing the selection & aggregation interface on a group-like
-    object sub-classes need to define: obj, exclusions
+    Mixin for the selection & aggregation interface on a group-like object.
+
+    Sub-classes need to define: obj, exclusions
     """
 
+    obj: Any
+    """Target object for the selection and aggregation."""
+    # GH 38239
+    # TODO obj here must be typed as FrameOrSeriesUnion,
+    # however this creates multiple mypy errors elsewhere.
+    # Those have to be addressed in a separate PR.
```

Comment:
I don't think this adds value. can remove

Reply:
Removed